reactioncommerce/cli
# feat: add create-project and develop commands for storefront

Signed-off-by: Brent Hoover <brent@thebuddhalodge.com>

This is the last major commands to add for this first phase:

1. Add the ability to have individual develop or create-project commands have specific dependencies in this case yarn for storefront. You don't need yarn unless you create storefront.

To test:

`./index.js create-project storefront mystore`

Then cd to the dir and do `npm install`

Then do `../index.js develop storefront`

This should start storefront in develop mode. Ctrl-C to stop
